### ISSUE TITLE
problems in R

### ISSUE THREAD

**Issue by JintaoSHOU** [nwfsc-assess/geostatistical_delta-GLMM#27]
Hi, James.
I am studying your spatiotemporal model lecture, when I compile the cpp file, the console of R has the following message
Error in compile("linear_model_v1.cpp") : Compilation failed
In addition: Warning message:
running command 'make -f "C:/PROGRA~1/R/R-34~1.3/etc/x64/Makeconf" -f "C:/PROGRA~1/R/R-34~1.3/share/make/winshlib.mk" -f "C:\Users\ADMINI~1\AppData\Local\Temp\Rtmp6DPT2d\file333c7d717945" SHLIB_LDFLAGS='$(SHLIB_CXXLDFLAGS)' SHLIB_LD='$(SHLIB_CXXLD)' SHLIB="linear_model_v1.dll" WIN=64 TCLBIN=64 OBJECTS="linear_model_v1.o"' had status 127 

I don't know why, looking forward your reply, thanks.

Jintao

**Comment by James-Thorson** [comment]
Status 127 indicates that TMB is not working, likely due to Rtools not being installed properly.  Please see https://github.com/nwfsc-assess/geostatistical_delta-GLMM/wiki/Steps-to-install-TMB, and either respond with an update or close the issue if that fixes it.

**Comment by JintaoSHOU** [comment]
Thanks for you reply, I solved this problem, but i got a new error

C:\Rtools\mingw_64\bin\nm.exe: linear_model_v1.o: File format not recognized
c:/Rtools/mingw_64/bin/g++ -shared -s -static-libgcc -o linear_model_v1.dll tmp.def linear_model_v1.o -LC:/PROGRA~1/R/R-34~1.3/bin/x64 -lR
linear_model_v1.o: file not recognized: File format not recognized
collect2.exe: error: ld returned 1 exit status
[1] 0
Thanks again

Jintao

**Comment by James-Thorson** [comment]
Did you confirm that TMB was working in the previous URL link?

**Comment by JintaoSHOU** [comment]
Yes, I got the 313.4137. I think I found the reason, I need  more time to confirm. Thank you, James
